lib/charms/tls_certificates_interface/v2/tls_certificates.py

```diff
@@ -1062,7 +1062,7 @@ def remove_certificate(self, certificate: str) -> None:
 
     def get_issued_certificates(
```

Comment:
Please be sure to fix the parts of the code that use this function and expect the previous output.
And the unit tests as well.

Reply:
Done